integration-tests/tests/test_utils/trait_mod.rs

```diff
@@ -0,0 +1,7 @@
+pub mod connection_router_trait;
+pub mod gateway_trait;
+pub mod multisig_prover_trait;
+pub mod multisig_trait;
+pub mod rewards_trait;
+pub mod service_registry_trait;
+pub mod voting_verifier_trait;
```

Comment:
Why do we need a file just for these? Why not just put these in mod.rs?

Reply:
i was trying not to over populate the original mod file. Will refactor.